tests/test_validate.py

```diff
@@ -25,7 +25,7 @@ def test_invalid_bundle():
         with open(bundle) as f:
             bundle = yaml.safe_load(f)
             bundle = unformat_bundle(bundle)
-            valid = ValidateCmd().validate(bundle)
+            valid, json = ValidateCmd().validate(bundle)
```

Comment:
Will be unittest for results dict added?

Reply:
Unit tests have been added in the latest commit.